docs/sql-guide/functions/function-coalesce.md

```diff
@@ -0,0 +1,116 @@
+---
+title: COALESCE()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# COALESCE() function
+
+`COALESCE()` function is used to return the first non-null value among the provided parameters.
+
+## Syntax
+
+```
+coalesce(value1, value2, ...)
+```
+
+## Arguments
+
+| Argument | Description | Length |
+|---|---|---|---|
+| `value1`,`value2`.. | Any expressions or values of same data type. The function returns the first non-null value among them. | 1 |
+
+## Returns
+
+The data type and value of the first non-null parameter
+
+## Examples
+
+### using Coalesce() function to handle null values
+
+```sql
```

Comment:
Just an FYI, I've setup an examples folder in the SQL guide because there are WAY too many table create/insert statements in the system that are inserted into function and data type pages, which was getting completely unwieldly.
What I'm trying to do is to have some basic tables that can be used for examples like this. It's a work in progress though :)